addons/web_editor/static/src/js/wysiwyg/widgets/link_popover_widget.js

```diff
@@ -50,6 +52,13 @@ export class LinkPopoverWidget {
         this.wysiwyg = params.wysiwyg;
         this.target = params.target;
         this.notify = params.notify;
+        this.debouncedNotify = debounce(
+            function (message, options) {
+                this.notify(message, options);
+            }.bind(this),
+            AUTOCLOSE_DELAY,
+            true
```

Comment:
If I am not wrong, "true" means "leading=false, trailing=true". With a big 3 seconds delay... it means that when we call debouncedNotify, we only show the notification 3 whole seconds later... am I right? Is that really ok?

(see next comment: I would like to test but not sure how to)

Reply:
> If I am not wrong, "true" means "leading=false, trailing=true"

@qsm-odoo I think it's the opposite
![image](https://github.com/user-attachments/assets/5bec79ca-af79-41d5-b8d4-1b0bdcc972ed)

Note: this API is awful...

Reply:
> if true, equivalent to exclusive leading. If false, equivalent to exclusive trailing.

Ok, I guess it's my english that is bad because I read that as "leading is excluded" instead of "only leading". But yeah, I would never pass a boolean ^^